### DISCUSSION TITLE
Steps for upgrade UDM SE firmware

### DISCUSSION THREAD

**Discussion by robbaman** [fabianishere/udm-iptv#216] [General]
Apologies for asking something that may already be explained somewhere, but I'm currently faced with my first firmware updates since I've started using my UDM SE on KPN so I'm a bit concerned that I break something and my wife will break me as a result.

I currently have a Network update pending (from 7.2.94 to 7.2.95) and a UniFi OS update from UDM SE 2.5.11 to 3.0.13.

Under the header 'Ensuring Installation across Firmware Updates' in the `Readme.md` it mentioned that `It is currently not possible to persist the installation across firmware updates` for the UniFi Dream Machine SE, but that 'Your configuration should remain, so only re-installation is necessary'...

Does this refer to updates for the UniFi OS or the Network application? 

And for the 're-installation', do I just use the same command again? `sh -c "$(curl https://raw.githubusercontent.com/fabianishere/udm-iptv/master/install.sh -sSf)"`?

Thanks in advance. IPTV and Internet are working now and I'd hate to break it by updating and not have any idea how to fix it.

**Comment by fabianishere** [comment]
> Does this refer to updates for the UniFi OS or the Network application?

Correct

> And for the 're-installation', do I just use the same command again?

Yes

**Reply by robbaman** [reply]
Thanks a bunch... I'll have a look then.
